.github/scripts/init_linux_port_vm_network.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+# This script install dependencies of service side of e2e tests and runs e2e test
+
```

Comment:
should this have those exit on failure preamble things that all your other scripts have